packages/website/src/routes/docs/daisy/pagination/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { $, component$, useStore } from '@builder.io/qwik';
+import { Pagination } from '@qwik-ui/headless';
+
+export default component$(() => {
+  const store = useStore({ page: 665 });
+
+  const incrementCount = $((newValue: number) => {
+    store.page = newValue;
+  });
+
+  return (
+    <div class="flex flex-col mt-4">
```

Comment:
This could be the daisy implementation here --> packages/daisy/src/components/pagination/pagination.tsx